tests/test_files/valid-multi-line_CHANGELOG.md

```diff
@@ -0,0 +1,3 @@
+## [Unreleased]
+  - Comment #1.
+  - Comment #2.
```

Comment:
think that should be without the space at the beginning of the line(and in all the other multilines)

Reply:
According to https://github.com/demisto/content/blob/master/docs/release_notes/README.md that's the format.